srcpkgs/mold/patches/00-undef-page-size.patch

```diff
@@ -0,0 +1,15 @@
+diff --git a/mold.h b/mold.h
+index 49fd7fd..9da6a9d 100644
+--- a/mold.h
++++ b/mold.h
+@@ -29,6 +29,10 @@
+ #include <vector>
+ #include <xxh3.h>
+ 
++#ifdef PAGE_SIZE
++#undef PAGE_SIZE
++#endif
```

Comment:
What does this patch do?

Reply:
@ericonr `musl`'s `limits.h` defines `PAGE_SIZE`, but `glibc`'s doesn't. `mold` author didn't know about that and assuming `PAGE_SIZE` is undefined, he named his variable the same way. That's one of the causes of `musl` build failing. This will be fixed in upstream surely, it's just a temporary workaround (this PR is still draft after all).

Reply:
Resolving because fixed upstream.